.npmrc.template

```diff
@@ -1,5 +0,0 @@
-registry=https://registry.yarnpkg.com/
-  
-@internxt:registry=https://npm.pkg.github.com
```

Comment:
I don’t think it’s necessary to remove the .npmrc.template, could be useful for new devs 🤔